docker-compose.yml

```diff
@@ -0,0 +1,49 @@
+version: "2.0"
+
+services:
+
+  scruid_druid:
+    image: mmartina/docker-druid:latest
+    container_name: scruid_druid
+    ports:
+      - "8081:8081"
+      - "8082:8082"
+      - "8083:8082"
+      - "8084:8082"
+
+  wait_for_druid:
```

Comment:
Maybe we can do this with a startup script as well. I'm hesitant to introduce another container for only waiting for the services to come up. Something similar to https://github.com/puckel/docker-airflow/blob/master/script/entrypoint.sh#L45-L57

WDYT?

Reply:
sure, I have updated `docker-compose.yml` and added `services.sh` script file that follows the approach that you suggest ([https://github.com/puckel/docker-airflow/blob/master/script/entrypoint.sh#L45-L57](https://github.com/puckel/docker-airflow/blob/master/script/entrypoint.sh#L45-L57)).